sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/README.md

```diff
@@ -22,6 +22,30 @@ See [Register app], [Grant scoped permission] for more information about web app
 ### Configure groups for sign in user
 In order to try the authorization action with this sample with minimum effort, [configure the user and groups in Azure Active Directory], configure the user with `group1`.
 
+### Configure AppRole for web application
+If you want to use `id_token` for authorization, we can use 
+`appRoles` feature of AAD to generate id_token's `roles` claim and then create `GrantedAuthority` from `roles` to implement access control. 
+Note the `roles` claim generated from `appRoles` is decorated with prefix `APPROLE`.
```

Comment:
1. `APPROLE` -> `APPROLE_`
2. Adjust the line breaks:
![image](https://user-images.githubusercontent.com/13167207/111404636-0fd56c80-870a-11eb-8092-2c3dc865072b.png)